server/config/database.js

```diff
@@ -9,6 +9,10 @@ export default () => ({
         migrations: {
             tableName: 'knex_migrations',
             directory: './server/migrations'
+        },
+        seeds: {
```

Comment:
Why do we have this here ? We are not using knex seeds in the traditional sense

Reply:
feedback implemented